Source/OxyPlot/Series/BoxPlotSeries.cs

```diff
@@ -94,6 +97,18 @@ public BoxPlotSeries()
         public double MedianThickness { get; set; }
 
         /// <summary>
+        /// Gets or sets the size of the mean point.
+        /// </summary>
+        /// <remarks>This property is only used when ShowMeanAsDot = true.</remarks>
+        public double MeanPointSize { get; set; }
```

Comment:
I think this should be consistent witht the marker type, size, color of the LineSeries/ScatterSeries etc.